connexion/decorators/response.py

```diff
@@ -44,10 +43,10 @@ def validate_response(self, data, status_code, headers):
             try:
                 # For cases of custom encoders, we need to encode and decode to
                 # transform to the actual types that are going to be returned.
-                data = json.dumps(data)
-                data = json.loads(data)
+                data = data.replace(b'\\"', b'"')
+                data = self.operation.api.jsonifier.loads(data)
```

Comment:
Probably we should add a top level method in the `self.operation` or `api` to handle this better. This `something.calling.another.internal.thing` will lead us to have a headache later.